sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -251,6 +288,10 @@ public HttpPipelinePolicy getRecordPolicy() {
      * @return {@link HttpPipelinePolicy} to record network calls.
      */
     public HttpPipelinePolicy getRecordPolicy(List<Function<String, String>> recordingRedactors) {
+        if (enableTestProxy) {
+            proxyVariableQueue.clear();
+            return startProxyRecording();
```

Comment:
Should we log a message here if `recordingRedactors` were passed that they won't have any effect on recording